src/frontends/common/include/openvino/frontend/frontend.hpp

```diff
@@ -143,6 +143,9 @@ class FRONTEND_API FrontEnd {
         add_extension(args...);
     }
 
+    /// \brief TODO: UPDATE
+    virtual void configure(const ov::AnyMap& config);
```

Comment:
I supposed we need to use existing API instead of new methods:
```c++

    template <typename... Types>
    inline InputModel::Ptr load(const Types&... vars) const {
        return load_impl({ov::Any{vars}...});
    }
    inline InputModel::Ptr load(const ov::AnyVector& vars) const {
        return load_impl(vars);
    }
```

Reply:
Done